Makefile

```diff
@@ -148,13 +148,15 @@ clean: clean-docs
 	find . -name '*.pyc' -type f -delete || true
 	find . -name __pycache__ -type d -delete || true
 	find . -name .pytest_cache -exec rm -rfv {} \; || true
-	cd frontend; rm -rf build node_modules
```

Comment:
This kind of doesn't matter but: why this change?

Reply:
for consistency. i found it easier to read (most of the other deletions are on their own line.) but happy to change it back!